### PR TITLE
7.62x25mm gun updates

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Reduced PPSH volume, added ranged_damage 2, range 2, AUTO 12, and revised the description.
Renamed the Tokarev to Tokarev tt-33, rewrote the item description, increased durability by 1, and decreased price by -$400.00.

The PPSH has a 33.2 inch total length, 10.6 inch barrel length, a 900 rpm rate of fire and 125-150 meter effective range. The Tokarev is durable, but it has grip issues.

### CODE REVIEW

**Comment by mugling** [comment]
Will merge conflict with #18362 (adjacent lines). Won't be difficult to rebase this PR though after the #18362 goes through.

**Comment by mugling** [comment]
The challenge is to somehow work the obscure calibers into the game.

Note `range` is pretty much a useless stat as you are limited by accuracy in almost all cases. A few items use it to limit range (eg. shotgun) but it doesn't serve much purpose here

**Comment by Zilenan91** [comment]
Does PPSH ammo even spawn? I've never seen any of it anywhere. Same goes with Kalashnikovs and their ammo, I've seen magazines for kalashnikovs but never the weapons themselves.

**Comment by mugling** [comment]
Yes, but it's (intentionally) rarer. Pawn shops, gun ranges and gun base safes often contain it. We've had previous PR's that increased the spawn rate so I don't really want to adjust it more. The original design specifications were (and remain) `9mm` and `223` everywhere, everything else less so.

Once handloading is fixed this become much less of a problem. It would be reasonable if the obscure guns could be made a little stronger to balance this. For example AKM should be exceptionally reliable etc. The obscure 32 ACP SMG is also gives an interesting option.

**Comment by BorkBorkGoesTheCode** [comment]
> Note  range  is pretty much a useless stat as you are limited by accuracy in almost all cases. A few items use it to limit range (eg. shotgun) but it doesn't serve much purpose here

Is the cx4 range buff going to be removed?

**Comment by mugling** [comment]
> Is the cx4 range buff going to be removed?

Its doesn't actually change the range, so yes

File: data/json/items/gun/762x25.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
     "id": "ppsh",
     "type": "GUN",
     "name": "PPSh-41",
-    "description": "The Soviet made PPSh-41, chambered in 7.62 Tokarev, provides a relatively large ammunition capacity coupled with low recoil and decent accuracy.",
+    "description": "The Soviet-made PPSh-41 is a mass-produced selective-fire submachine gun. It has a relatively high rate of fire.",
```
Comment (Night-Pryanik) [inline data/json/items/gun/762x25.json]: Two spaces after period, please.

**Comment by BorkBorkGoesTheCode** [comment]
> The challenge is to somehow work the obscure calibers into the game.

Ammo_obscure could be added to gunsmiths.

**Comment by mugling** [comment]
> Ammo_obscure could be added to gunsmiths.

No. The whole purpose is to have them be obscure and that spawn wouldn't help much anyway.

What I mean is the obscure guns need to be 'worth' the rarity of the their ammo

**Comment by mugling** [comment]
I've merged this and rebased #18362 because I see you struggling with rebase in your other PR. In future though try not to open PR's that will conflict with larger work that is already in progress. Now the PR queue is on one page it's much easier to check what everyone is actively working on.

**Comment by BorkBorkGoesTheCode** [comment]
> I've merged this and rebased #18362 because I see you struggling with rebase in your other PR. In future though try not to open PR's that will conflict with larger work that is already in progress. Now the PR queue is on one page it's much easier to check what everyone is actively working on.

Thank you. I apologize for wasting time.

**Comment by mugling** [comment]
Providing you improve your skills with each PR there is no wastage

**Pull request merged**